coalib/core/Bear.py

```diff
@@ -130,6 +132,21 @@ class Bear:
     CAN_FIX = set()
     ASCIINEMA_URL = ''
     BEAR_DEPS = set()
+    """
+        Attributes for a new bear.
+
+        :param section:
+            The section object where bear settings are contained.
+        :param file_dict:
+            The file-dictionary containing a mapping of filenames to the
+            according file contents.
+        :raises RuntimeError:
+            Raised when bear requirements are not fulfilled.
+    """
+    section = attr.ib(hash=False)
+    file_dict = attr.ib(hash=False)
+    _dependency_results = attr.ib(init=False, default=defaultdict(list),
+                                  hash=False)
```

Comment:
@Makman2 As Lasse said , even though the usecases might be limited or tricky as you mentioned.It's a good idea to store all data members which are holding important data as an Attribute. So that the `attrs.as_dict` of the entire core repo will contain the important data members.This can be a starting point to improve our Data Model. Distinguishing between data_members is the most important aspect of `attrs` , other features have been implemented in some way or the other in the repo already.